src/assets/styles/_loader.scss

```diff
@@ -1,46 +1,45 @@
 .loader-wrapper {
```

Comment:
This file is correct and does not require any changes. We can have nested scss classes. Please remove the changes from this file.